tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+	"compilerOptions": {
+		"lib": [
+			"es5",
+			"es6",
+			"dom",
+			"es2015.core",
+			"es2015.collection",
+			"es2015.generator",
+			"es2015.iterable",
+			"es2015.promise",
+			"es2015.proxy",
+			"es2015.reflect",
+			"es2015.symbol",
+			"es2015.symbol.wellknown",
+			"esnext.asynciterable"
+		],
+		"module": "commonjs",
+		"target": "es6",
+		"jsx": "react-native",
+		"strict": true,
+		"moduleResolution": "node",
+		"sourceMap": false,
```

Comment:
No source maps with TS? Wanted behavior?

Reply:
This was related to a test but this should be true